app/post/shared/post-actions.directive.js

```diff
@@ -31,6 +36,20 @@ function (
                     $location.path('/views/list');
                 });
             };
+
+            $scope.edit = function (post) {
+                $location.path('/posts/' + post.id + '/edit');
+            };
+
+            $scope.updateStatus = function (status) {
```

Comment:
I think this is a good candidate to move into the post-actions service? or post edit service? I assume the logic is shared in a few places where we change status..